CSharp_Net_module1_2_3_lab/CSharp_Net_module1_2_3_lab/Program.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharp_Net_module1_2_3_lab
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            // 10) declare 2 objects
+            var uah1 = new Money(CurrencyTypes.UAH, 120.5);
+            Console.WriteLine("Created uah1: " + uah1);
+
+            var uah2 = new Money(CurrencyTypes.UAH, 12.38);
+            Console.WriteLine("Created uah2: " + uah2);
+
+            // 11) do operations
+            // add 2 objects of Money
+            var uah3 = uah1 + uah2;
+            Console.WriteLine("uah3 = uah1 + uah2 = " + uah3);
+
+            // add 1st object of Money and double
+            var uah4 = uah3 + 17.35;
+            Console.WriteLine("uah4 = uah3 + 17.35 =" + uah4);
+
+            // decrease 2nd object of Money by 1 
+            uah2--;
+            Console.WriteLine("uah2-- = " + uah2);
+
+            // increase 1st object of Money
+            var uah5 = uah1 * 3;
+            Console.WriteLine("uah5 = uah1 * 3 = " + uah5);
+
+            // compare 2 objects of Money
+            Console.WriteLine("uah1 > uah2 =" + (uah1 > uah2).ToString());
```

Comment:
You don't need to invoke ToString method here

Reply:
Yes, but I think it not bad way, because I just used explicit convert to string instead implicit you mean. And in both cases conversation will be execute because this expression (uah1 > uah2) returns bool that concat to string.